api/src/main/java/eu/europeana/fulltext/search/web/FTSearchController.java

```diff
@@ -96,7 +98,10 @@ public ResponseEntity searchIssue(@PathVariable String datasetId, @PathVariable
         String searchId = request.getRequestURI() + "?" + request.getQueryString();
         SearchResult searchResult = searchService.searchIssue(searchId, new EuropeanaId(datasetId, localId), qry,
                 pageSize, annoTypes, requestVersion, (debug != null));
-        return new ResponseEntity<>(searchResult, HttpStatus.OK);
+
+        return ResponseEntity.status(HttpStatus.OK)
+                .header(HttpHeaders.CONTENT_TYPE, HttpHeaders.CONTENT_TYPE_JSONLD)
+                .body(searchResult);
```

Comment:
I propose to also reuse AcceptUtils.addContentTypeToResponseHeader()  here, so we include IIIF version.

Reply:
done. But have separated the json and jsonLd methods.

Reply:
done. But added separate method for json and jsonLd